object.js

```diff
@@ -50,7 +50,7 @@ const tonic = projects[0]; // Obtiene el proyecto "Tonic" del arreglo
 
 const tonicHTML = `
   <article>
-    <img src="${tonic.image}" alt="${tonic.name}">
+    
     <div class="adjust">
       <div class="leftblock">
         <h2>${tonic.name}</h2>
```

Comment:
- [ ] You did put in effort, however, here are few things to note:
- Your popup is only for the first card, as I noticed `tonic` was declared for the first index of your card's array. Kindly implement this for all other cards by possibly using a `map` to render all 👍 
- The cards do not have a close button when rendered and have to be refreshed to be deleted. please fix this to give a better user experience, and also to follow project guide lines 👍 

## Instruction:
![image](https://user-images.githubusercontent.com/50990925/224521282-0ae5988d-295e-4813-9e76-4c137650488e.png)

## Your view:
![image](https://user-images.githubusercontent.com/50990925/224521299-45208f81-64e2-49a9-a90b-61405eea9e12.png)